packages/expo-cli/src/commands/installAsync.ts

```diff
@@ -141,7 +141,7 @@ export async function actionAsync(
   await packageManager.addWithParametersAsync(versionedPackages, parameters);
 
   try {
-    exp = getConfig(projectRoot, { skipSDKVersionRequirement: true, skipPlugins: true }).exp;
+    exp = getConfig(projectRoot, { skipSDKVersionRequirement: true, skipPlugins: false }).exp;
```

Comment:
```suggestion
    exp = getConfig(projectRoot, { skipSDKVersionRequirement: true }).exp;
```
Also ensure the functionality [mentioned here](https://github.com/expo/expo/pull/17936#pullrequestreview-1014712458) continues to work.

Reply:
I prefer to be explicit, but I'll commit the suggestion.